hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestReadRetries.java

```diff
@@ -147,7 +145,8 @@ public void testPutKeyAndGetKeyThreeNodes()
             ReplicationFactor.THREE, new HashMap<>());
     KeyOutputStream groupOutputStream =
         (KeyOutputStream) out.getOutputStream();
-    XceiverClientManager manager = groupOutputStream.getXceiverClientManager();
+    XceiverClientManager manager =
+        (XceiverClientManager) groupOutputStream.getXceiverClientFactory();
```

Comment:
Why cast instead of changing the variable to `XceiverClientFactory`?

Reply:
Both can work, but I feel it more safe as it doesn't require changing method signatures in `KeyOutputStream`. Even if `getXceiverClientFactory` is marked as `@VisibleForTesting` I feel it safer to return only the interface. Unit test can have cast, it's expected to have access to the internal representations.

Reply:
I meant: is there anything against changing to

```java
    XceiverClientFactory manager =
        groupOutputStream.getXceiverClientFactory();
```

Sorry if it was unclear.

Reply:
Aghh, got it. Obviously: you are right, I just didn't notice the usage of the `manager`. Pushed the improved version.